lib/core/filtering/simple_rule.rb

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+require 'utility/logger'
+
+module Core
+  module Filtering
+    class SimpleRule
+      POLICY = 'policy'
```

Comment:
I don't mind using constants as hash keys but I wonder why this is so in you case.

Reply:
It's your fault. I was trying to avoid symbols, but also avoid risking typos 😅 

See also: https://github.com/elastic/connectors-ruby/pull/412#discussion_r1021744506
WDYT?

Reply:
I've switched to using string literals in the follow-up PR